File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/specification/CsChapterItem.java
package com.philips.cdp.prxclient.datamodels.specification;

import java.io.Serializable;
import java.util.List;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class CsChapterItem implements Serializable {

	private static final long serialVersionUID = 5009770779919717578L;
	@SerializedName("csChapterCode")
	@Expose
	public String csChapterCode;

	@SerializedName("csItem")
	@Expose
	public List<CsItemItem> csItem;

	@SerializedName("csChapterName")
	@Expose
	public String csChapterName;

	@SerializedName("csChapterRank")
	@Expose
	public String csChapterRank;
}

File: Source/prx/Source/Library/PrxSample/prx/src/main/java/com/philips/cdp/prxclient/datamodels/specification/CsChapterItem.kt
package com.philips.cdp.prxclient.datamodels.specification

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class CsChapterItem(
        var csChapterCode: String? = null,
        var csItem: List<CsItemItem>? = null,
        var csChapterName: String? = null,
        var csChapterRank: String? = null
) : Parcelable